presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkMutableRowPageInput.java

```diff
@@ -39,7 +40,7 @@
 {
     private static final int TARGET_SIZE = 1024 * 1024;
     private static final int BUFFER_SIZE = (int) (TARGET_SIZE * 1.2f);
-    private static final int MAX_ROWS_PER_ZERO_COLUMN_PAGE = 10_000;
+    private static final int MAX_ROWS_PER_PAGE = 20_000;
```

Comment:
curious why 20_000 will be better than 10_000 ? ;)

Reply:
I'm going to use if for both now, zero columns an non zero columns pages. 20_000 seem to be better (e.g.: for a single bigint column page the size will be ~160kb instead of 80kb). We usually try to keep the page size above 100kb.